### PR TITLE
Updated Funding Page

### PR DESCRIPTION
**Pull request by KimberleyChong**
**Description**
Updates to the funding page:

- New funders: Amazon and NHGRI
- Separation of "Current" and "Previous" funders
- Overall UI refresh

**Before:**
old_fundingpage

**After:**
new_funding

Amazon and NHGRI are also added as Funders on the About page:
image
Funders no longer fit on one line... (let me know if yo have any thoughts on this) Shrinking the size makes them look microscopic

**Review Instructions**
Ensure information looks correct

**Issue**
[DOCK-2310](https://ucsc-cgl.atlassian.net/browse/DOCK-2310)
[GitHub Issue
](https://github.com/dockstore/dockstore/issues/5328)
**Security**
If there are any concerns that require extra attention from the security team, highlight them here.

Please make sure that you've checked the following before submitting your pull request. Thanks!

- [x] Check that your code compiles by running `npm run build`
- [x] Ensure that the PR targets the correct branch. Check the milestone or fix version of the ticket.
- [x] If this is the first time you're submitting a PR or even if you just need a refresher, consider reviewing our [style guide](https://github.com/dockstore/dockstore/wiki/Dockstore-Frontend-Opinionated-Style-Guide#pr-checklist)
- [x] Do not bypass Angular sanitization (bypassSecurityTrustHtml, etc.), or justify why you need to do so
- [x] If displaying markdown, use the `markdown-wrapper` component, which does extra sanitization
- [x] Do not use cookies, although this may change in the future
- [x] Run `npm audit` and ensure you are not introducing new vulnerabilities
- [x] Do due diligence on new 3rd party libraries, checking for CVEs
- [x] Don't allow user-uploaded images to be served from the Dockstore domain
- [x] If this PR is for a user-facing feature, create and link a documentation ticket for this feature (usually in the same milestone as the linked issue). Style points if you create a documentation PR directly and link that instead.
- [x] Check whether this PR disables tests. If it legitimately needs to disable a test, create a new ticket to re-enable it in a specific milestone. 

[DOCK-2310]: https://ucsc-cgl.atlassian.net/browse/DOCK-2310?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/app/funding/funding.component.html [hunk 1]
```diff
@@ -1,33 +1,39 @@
...
-    <div *ngFor="let funder of funders" class="col-xs-12 col-md-6 col-lg-6">
-      <mat-card class="my-3">
-        <mat-card-header>
-          <mat-card-title>{{ funder.title }}</mat-card-title>
-        </mat-card-header>
-        <div class="image-wrapper">
-          <a [href]="funder.website">
-            <img [src]="funder.imageSource" [alt]="funder.altImageText" class="center-block funding-img" />
-          </a>
+    <div *ngFor="let section of fundingDisplay" class="">
```
Comment (kathy-t) [inline src/app/funding/funding.component.html:7]: empty class

**Review by kathy-t — APPROVED** [review]

**Comment by denis-yuen** [comment]
> I can't request a review from @bvizzier for some reason... maybe permissions?

think we centralised permissions through the teams at some point and @bvizzier was left out, re-adding

**Comment by denis-yuen** [comment]
> Funders no longer fit on one line... (let me know if yo have any thoughts on this) Shrinking the size makes them look microscopic

AWS and genome canada look too close, as if almost related to each other. Maybe we could use actual text on a second line for BD Catalyst. I don't know if that will be enough to get things on one line

**Review by denis-yuen — COMMENTED** [review]
one thing to experiment with

**Comment by KimberleyChong** [comment]
> one thing to experiment with

Okay, I shrank both the padding and images, better now (just a little smaller...):
image

**Comment by denis-yuen** [comment]
Thanks!
Now, the fact that the three NIH funders are not consecutive and have different sized "NIH" badges makes me feel a little funny, but I don't know if fixing it is going overboard :thinking:

**Comment by KimberleyChong** [comment]
> Now, the fact that the three NIH funders are not consecutive and have different sized "NIH" badges makes me feel a little funny, but I don't know if fixing it is going overboard

Unfortunately each one is proportioned differently... but I have an idea 😉

**Comment by KimberleyChong** [comment]
> Now, the fact that the three NIH funders are not consecutive and have different sized "NIH" badges makes me feel a little funny, but I don't know if fixing it is going overboard
```
// Normalize image sizes for NIH logos
.funders[alt="NIH Logo"], .funders[alt="BISTI Logo"], .funders[alt="NHGRI Logo"] {
  height: 1.7rem;
}
```
They're  _fairly_ similar in size now
image

File: src/app/funding/funding.component.ts [hunk 2]
```diff
@@ -63,13 +68,44 @@ export class FundingComponent {
...
-    this.NIHFunder,
     this.BISTIFunder,
+    this.NIHFunder,
+    this.NHGRIFunder,
     this.BioDataCatalyst,
     this.CFIFunder,
+    this.AmazonFunder,
+  ];
+
+  currentFunders: Funder[] = [this.AmazonFunder, this.GenomeCanadaFunder, this.NHGRIFunder, this.NIHFunder, this.ProvinceOfOntarioFunder];
```
Comment (coverbeck) [inline src/app/funding/funding.component.ts:97]: They are no longer a current funder, correct @bvizzier ?

**Review by coverbeck — APPROVED** [review]
I left one question in there; I hope it doesn't throw off the layout too much if I'm right. :)

File: src/app/funding/funding.component.ts (hunk 2 above)
Comment (coverbeck) [reply src/app/funding/funding.component.ts:97]: AWS. Sorry, It was in my mind, but I forgot to write it down.

**Review by bvizzier — COMMENTED** [review]
Please check to see if there are options to make the NIH & NHGRI logos more readable.  As previously mentioned, maybe adding a new line with different scaling?

**Comment by KimberleyChong** [comment]
> Please check to see if there are options to make the NIH & NHGRI logos more readable. As previously mentioned, maybe adding a new line with different scaling?

Alright, we now have two lines with all the NIH logos together on the second row:
image

Best solution I found was to use the `slice` pipe to display the first and second halves in two rows:
https://github.com/dockstore/dockstore-ui2/blob/a1eba62926ce3bdea56aacbfcfeb43aede3d8fe5/src/app/about/about.component.html#L234

https://github.com/dockstore/dockstore-ui2/blob/a1eba62926ce3bdea56aacbfcfeb43aede3d8fe5/src/app/about/about.component.html#L239

**Review by bvizzier — APPROVED** [review]

**Review by denis-yuen — APPROVED** [review]

File: src/app/funding/funding.component.ts (hunk 2 above)
Comment (coverbeck) [reply src/app/funding/funding.component.ts:97]: Sorry, what is the resolution to this? Do we keep them in currentFunders or not?

**Review by coverbeck — CHANGES_REQUESTED** [review]
Temporarily blocking until funders issue is resolved one way or another.

**Comment by KimberleyChong** [comment]
> Sorry, what is the resolution to this? Do we keep them in currentFunders or not?

Amazon is a current funder right? @bvizzier

**Review by coverbeck — APPROVED** [review]

**Review by svonworl — APPROVED** [review]

**Pull request merged**